internal/keygenext/release.go

```diff
@@ -120,9 +120,10 @@ func (r *Release) Get() error {
 	// TODO(ezekg) Add support for custom query params to SDK
 	type querystring struct {
 		Package string `url:"package,omitempty"`
+		Product string `url:"product,omitempty"`
 	}
 
-	qs := querystring{Package: *r.PackageID}
+	qs := querystring{Package: *r.PackageID, Product: r.ProductID}
```

Comment:
I think you can actually just replace this with `Product: Product`, since the current product is already defined as a global in the `keygenext` package (see it being used above in `keygen.ClientOptions`).